website_sale_resource_booking/tests/test_ui.py

```diff
@@ -40,9 +51,9 @@ def test_checkout(self):
         self.start_tour(
             "/shop?search=test not bookable product",
             "website_sale_resource_booking_checkout",
-        )
-        # Find Mr. A's cart
-        so = self.env["sale.order"].search([("partner_id", "=", "Mr. A")])
+            login="ptl",
```

Comment:
Why changing the test?

Reply:
The test was in conflict with the website_require_login module. To solve that, a user is created and logged in. The behavior is the same as if it were being performed without being logged in.

Reply:
Ok, anyway give it a more comprehensive or less generic name and login or you could clash again against other module's data